src/services/identity/__tests__/CollaboratorsService.spec.ts

```diff
@@ -286,7 +315,10 @@ describe("CollaboratorsService", () => {
       // Arrange
       collaboratorsService.deriveAllowedRoleFromEmail = (jest.fn(
         () => CollaboratorRoles.Admin
-      ) as unknown) as () => Promise<CollaboratorRoles | null>
+      ) as unknown) as () => Promise<Exclude<
```

Comment:
shall we just create a nonIsomerAdmin role type since we are using this at multiple places?

Reply:
Adjusted in 771d9868!